src/eko/matching_conditions/n3lo/s_functions.py

```diff
@@ -0,0 +1,801 @@
+# -*- coding: utf-8 -*-
+"""
+This module contains some additional harmonics sum.
+Defintion are coming from :cite:`MuselliPhD,Bl_mlein_2000,Blumlein:2009ta`
+"""
+import numba as nb
+import numpy as np
+
+from eko.anomalous_dimensions import harmonics
+
+from . import f_functions as f
+from . import g_functions as gf
+
+zeta2 = harmonics.zeta2
+zeta3 = harmonics.zeta3
+zeta4 = harmonics.zeta4
+zeta5 = harmonics.zeta5
+
+li4half = 0.517479
+log2 = np.log(2)
+
+
+@nb.njit("c16(c16)", cache=True)
+def harmonic_Sm1(N):
```

Comment:
I'd suggest to move the simple S_{-n} into the  `ad.harmonics` package since we need to reuse some of them for the ad - maybe we should even open a dedicated subpackage `eko/harmonics` with something like `single_letter`, containing `s_1,s_m1,...`, `tow_letter` with `s_21`, etc.

Reply:
yes indeed the separation by weights is a bit more convenient, since the `w+1` can be sometimes contain some `w` sums ...